change/@fluentui-web-components-33ee01f8-ba19-4b46-80cc-ec22b85b7483.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "update high contrast styles",
```

Comment:
@khamudom can you update this comment? I think this addresses more than just the high contrast styles :)

Reply:
thanks! updated